docs/docsite/rst/network_getting_started.rst

```diff
@@ -0,0 +1,246 @@
+.. network-getting-started-example:
+
+*******************************
+Network getting started example
+*******************************
+
+.. contents:: Topics
+
+
+Overview
+========
+
+Objective
+---------
+
+I want to understand how Ansible can be used to connect to multiple network devices.
+
+.. FIXME FUTURE Gundalow - Link to examples index once created
+
+.. note:: Examples, not production code
+   This is not for production use, it's more to demonstrate the different areas that need to be considered and provide a foundation level knowledge.
+
+Prerequisites
+-------------
+
+* Ansible 2.3 (or higher) installed :doc:`intro_installation`
+* One or more network device compatible with Ansible
+* Basic understanding of YAML :doc:`YAMLSyntax`
+* Basic understanding of Jinja2 Templates :doc:`playbooks_templating`
+
+.. FIXME FUTURE Gundalow - Once created we will link to the connection table here (which platforms support network_cli & credentials through inventory)
+.. FIXME FUTURE Gundalow -  Using ``ansible_ssh_pass`` will not work for REST transports such as ``eapi``, ``nxapi`` - Once documented in above FIXME add details her
+
+Audience
+--------
+
+* Basic Linux (comfortable on the command line)
+* Basic Network switch & router configuration knowledge
+
+
+Solution
+=========
+
+Create an inventory file.
+
+
+**Inventory file**
+
+Create a file called ``inventory``, containing:
+
+.. code-block::
+
+   [switches:children]
+   ios
+   vyos
+
+   [ios]
+   ios01.example.net ansible_connection=local ansible_user=cisco ansible_ssh_pass=cisco
+
+   [vyos]
+   vyos01.example.net ansible_connection=local ansible_user=admin ansible_ssh_pass=admin
+
+
+**Playbook**
+
+Create a file called ``fetch-facts.yml`` containing the following:
+
+.. code-block:: yaml
+
+   - name: "Download switch configuration"
+     hosts: switches
+     gather_facts: no
+
+     tasks:
+       ###
+       # Collect data
+        - name: Gather facts (ios)
+          ios_facts:
```

Comment:
Can we talk about the basic 3 modules, command, config and facts?  This will help them in understanding, basic configuration, retrieving the show command and facts from Ansible. This is what they typically do on a day to day basis manually.

Reply:
I was going to detail `_command` and `_config` in separate pages.

I should go into extra detail here on how the `_facts` modules, work and the `ansible_net*l` variables that get populated.

Reply:
got it